src/main/scala/subsystem/HasTiles.scala

```diff
@@ -10,13 +10,18 @@ import freechips.rocketchip.devices.tilelink.{BasicBusBlocker, BasicBusBlockerPa
 import freechips.rocketchip.diplomacy._
 import freechips.rocketchip.diplomaticobjectmodel.logicaltree.{LogicalModuleTree}
 import freechips.rocketchip.interrupts._
-import freechips.rocketchip.tile.{BaseTile, LookupByHartIdImpl, TileParams, HasExternallyDrivenTileConstants, InstantiableTileParams, PriorityMuxHartIdFromSeq}
+import freechips.rocketchip.tile.{BaseTile, LookupByHartIdImpl, TileParams, InstantiableTileParams, MaxHartIdBits}
 import freechips.rocketchip.tilelink._
 import freechips.rocketchip.util._
 
 /** Entry point for Config-uring the presence of Tiles */
 case class TilesLocated(loc: HierarchicalLocation) extends Field[Seq[CanAttachTile]](Nil)
 
+/** Whether a global hart id prefix is going to be dynamically applied to the static tile hart ids */
+case object HartPrefixKey extends Field[Boolean](false)
+case object SubsystemExternalHartIdWidthKey extends Field[Option[Int]](None)
+case object SubsystemExternalResetVectorKey extends Field[Boolean](true)
```

Comment:
```suggestion
/** Whether the Subsystem module has I/Os that control the tile's power-on reset vectors
case object SubsystemExternalResetVectorKey extends Field[Boolean](true)
```

Reply:
Adding some more complicated comments here in a local commit